src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap");
+
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 0;
+  }
+
+  &__product-image {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  &__title {
+    padding-top: 40px;
+    width: 166;
+    height: 36;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    color: $secondary;
+    padding: 4px 0 0;
+    line-height: 14px;
+  }
+
+  &__button {
+    all: unset;
+    box-sizing: border-box;
+    display: flex;
+    width: 166px;
+    height: 40px;
+    background-color: $blue-accent;
+    cursor: pointer;
+    border-radius: 5px;
+
+    font-size: 14px;
+    color: #fff;
+    font-weight: 700;
+    text-transform: uppercase;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background-color: #fff;
+      color: $blue-accent;
+      border: 1px solid $blue-accent;
```

Comment:
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on `:hover`